src/test/java/com/google/alpollo/AnalysisTest.java

```diff
@@ -124,11 +125,11 @@ public void top10SalientEntitiesWithLessThan10Entities() throws IOException {
     String responseString = responseWriter.toString();
 
     List<SongEntity> actual = gson.fromJson(responseString, new TypeToken<List<SongEntity>>(){}.getType());
-    List<SongEntity> expected = Arrays.asList(new SongEntity("mountain", 0.84, "OTHER", EMPTY_STRING),
-        new SongEntity("mountain", 0.06, "LOCATION", EMPTY_STRING),
-        new SongEntity("story", 0.05, "WORK_OF_ART", EMPTY_STRING),
-        new SongEntity("sale", 0.03, "OTHER", EMPTY_STRING),
-        new SongEntity("tale", 0.01, "WORK_OF_ART", EMPTY_STRING));
+    List<SongEntity> expected = Arrays.asList(
+        new SongEntity("mountain", 0.84, new HashSet<String>(Arrays.asList("OTHER", "LOCATION")), EMPTY_STRING),
```

Comment:
Shouldn't we also combine the salience here? In this case I'd expect to see  `{ "mountain", ["OTHER", "LOCATION"], 0.90 }`

Reply:
That's actually a great idea, we should add the saliences too.

Reply:
@BudiGabe don't forget to change frontend part in this case and display according to the new format of response. Or I can do it if you want.

Reply:
The response should be the same. What do you think there is to change?

Reply:
I just checked how it displays list of types, looks ok